src/sounds.cpp

```diff
@@ -93,6 +98,21 @@ void sounds::add_footstep( const tripoint &p, int volume, int, monster * )
         std::make_pair( p, sound_event {volume, "", false, true, "", ""} ) );
 }
 
+const char * sounds::sound_event_id( verbal_shout_id verbal_shout_id ) {
```

Comment:
I'm pretty sure returning a pointer to a local, non-static C string is undefined behavior.

You could make static local variables for all the used strings and return those, though.